packages/scm/src/browser/style/index.css

```diff
@@ -13,6 +13,14 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
+
+.theia-ScmCommit {
```

Comment:
please follow up-to-date guidance for new css classes: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css-use-lower-case-with-dashes

Reply:
now renamed.